data-exporter/src/main/java/org/reactome/release/dataexport/NCBIEntry.java

```diff
@@ -68,8 +68,9 @@ public String getUniprotDisplayName() {
 				"MATCH (rgp:ReferenceGeneProduct)-[:referenceDatabase]->(rd:ReferenceDatabase)",
 				"MATCH (rgp)-[:referenceGene]->(rds:ReferenceDNASequence)",
 				"WHERE rd.displayName = 'UniProt' AND rds.databaseName = 'NCBI Gene'",
-				"RETURN rgp.dbId, rgp.displayName, rgp.identifier, rds.identifier",
-				"ORDER BY rgp.identifier;"
+				"RETURN rgp.dbId, rgp.displayName, coalesce(rgp.variantIdentifier, rgp.identifier) as rgp_accession,"
```

Comment:
Based on what I found online, the coalesce method will return the first shared value between the submitted expression list? https://neo4j.com/docs/cypher-manual/current/functions/scalar/#functions-coalesce

Reply:
Yup, `coalesce` is used here to take the "variant identifier" value, which is the UniProt isoform accession, if it exists, but to use the "identifier" which is the parent/canonical UniProt accession otherwise for non-isoforms where "variant identifier" will be null.